projects/vdk-control-cli/src/vdk/internal/control/utils/control_utils.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+from configparser import ConfigParser
+from configparser import MissingSectionHeaderError
+
+from vdk.internal.control.exception.vdk_exception import VDKException
+
+
+log = logging.getLogger(__name__)
+
+
+def read_job_configuration(
+    config_parser: ConfigParser, configuration_file_path: str
+) -> None:
+    """
+    Read the Data Job config.ini file.
+
+    :param config_parser: ConfigParser instance to be used for reading the
+    configuration file.
+    :param configuration_file_path: Path of the config.ini file
+    """
+    try:
+        config_parser.read(configuration_file_path)
+    except (MissingSectionHeaderError, Exception) as e:
+        log.debug(e, exc_info=True)  # Log the traceback in DEBUG mode.
+        raise VDKException(
+            what="Cannot parse the Data Job configuration file.",
```

Comment:
Which file cannot be parsed ? Add `configuration_file_path` in the message.

```suggestion
            what=f"Cannot parse the Data Job configuration file {configuration_file_path}.",
```

Reply:
Added, thanks